release/platform/platform.go

```diff
@@ -34,7 +34,11 @@ const (
 type Arch string
 
 const (
-	ArchArm64   Arch = "arm64"
+	ArchArm64 Arch = "arm64"
+	// While arm64 and aarch64 are the same architecture, some Linux distros
```

Comment:
I did much more testing, and `aarch64` packages will install on an `arm64` platform. I am not sure what I was seeing anymore in previous tests, but the conclusion is definitive - `aarch64` and `arm64` are aliases on all platforms. What this boils down to is consistency in where to find packages, not what will install.

Reply:
Thanks, I updated this comment.